docs/configuration.md

```diff
@@ -0,0 +1,53 @@
+# Configuration
+
+## Configuration methods
+
+## Global settings
+
+| Environment variable        | Description                                                                | Default value |
+|-----------------------------|----------------------------------------------------------------------------|---------------|
+| `OTEL_GO_AUTO_TARGET_EXE`   | Sets the binary for the Go application to be instrumented.                 |               |
+| `OTEL_GO_AUTO_GLOBAL`       | Enables the OpenTelemetry global implementation.                          |               |
```

Comment:
Here is a more accurate description: https://github.com/open-telemetry/opentelemetry-go-instrumentation/blob/d730a91aaa5485951d9f8b3883b64936d021e693/cli/main.go#L62